generate-attribution/generate-attribution-file.js

```diff
@@ -338,6 +340,23 @@ async function populateVersionAndModuleFromDep(dependencies) {
     return finalDeps;
 }
 
+function generateAuthorizationHeader() {
+    const githubTokenFile = "/secrets/github-secrets/token";
+    fs.stat(githubTokenFile, (error, stats) => {
```

Comment:
Sorry this is the pattern to follow, in the catch block return empty object, that means file doesn't exist

https://github.com/aws/eks-distro-build-tooling/blob/ac2191b893bfde5628e758007c7fda5306cd7e36/generate-attribution/generate-attribution-file.js#L220

Reply:
Thanks @jaxesn, updated the PR accordingly.